frontend/catalyst/api_extensions/differentiation.py

```diff
@@ -247,12 +247,11 @@ def workflow(x):
             def circuit(x):
                 qml.RX(jnp.pi * x, wires=0)
                 return qml.expval(qml.PauliY(0))
+            return value_and_grad(circuit)(x)
 
-            g = value_and_grad(circuit)
-            return g(x)
-
-    >>> workflow(2.0)
-    (array(0.2), array(-3.14159265))
```

Comment:
this was not the correct value, there must have been a copy-paste issue with this docstring. I verified the new values against non-qjit PL